code/modules/mob/living/carbon/human/human_stripping.dm

```diff
@@ -70,6 +87,43 @@ GLOBAL_LIST_INIT(strippable_human_items, create_strippable_list(list(
 	mob_source.update_worn_undersuit()
 	mob_source.update_body()
 
+/datum/strippable_item/mob_item_slot/jumpsuit/proc/do_adjust_sensor(atom/source, mob/user, obj/item/clothing/under/jumpsuit)
+	if(!jumpsuit.has_sensor)
+		return
+
+	var/static/list/sensor_mode_text_to_num = list(
+		"Off" = SENSOR_OFF,
+		"Living" = SENSOR_LIVING,
+		"Vitals" = SENSOR_VITALS,
+		"Tracking" = SENSOR_COORDS,
+	)
+	var/static/list/senor_mode_num_to_text = list( // keep this as the inverse of the above list
+		"[SENSOR_OFF]" = "Off",
+		"[SENSOR_LIVING]" = "Living",
+		"[SENSOR_VITALS]" = "Vitals",
+		"[SENSOR_COORDS]" = "Tracking",
+	)
+
+	var/new_mode = tgui_input_list(user, "Set To What", "Adjust Sensors", sensor_mode_text_to_num, senor_mode_num_to_text["[jumpsuit.sensor_mode]"])
+	new_mode = sensor_mode_text_to_num[new_mode]
+	if(isnull(new_mode)) // also catches returning null
+		return
+
+	if(!user.Adjacent(source))
+		source.balloon_alert(user, "can't reach!")
+		return
+
+	to_chat(source, span_notice("[user] is trying to adjust your [jumpsuit]'s sensor."))
```

Comment:
Probably balloon alert?

Reply:
needs to be logged to chat so players can audit "why is my shit changed"